device/WakeOnLan/controller.js

```diff
@@ -0,0 +1,130 @@
+﻿'use strict';
+
+const BluePromise = require('bluebird');
+const fs = require('fs')
+const os = require('os');
+const net = require('net');
+const dgram = require('dgram');
+
+const PORT_NBR = 9;
+const DEFAULT_IPADDRESS = '255.255.255.255';
+
+let wolMap = "wol.map";
```

Comment:
Also - let is appropriate here since I can potentially change the name in the setMapPath function (although that reminds me I need to change the name of the variable since it's a path to the map file)